examples/config.yml

```diff
@@ -9,8 +9,9 @@ server:
   actionCacheReadOnly: false
   port: 8980
   grpcMetrics:
-    enabled: false
-    provideLatencyHistograms: false
+    enabled: true
+    provideLatencyHistograms: true
+    latencyBuckets: [0.001, 0.005, 0.01, 0.05, 0.075, 0.1, 0.25, 0.5, 1.0, 2.0, 5.0, 10.0]
```

Comment:
we still get the infinite bucket, right? i also suggest example to include buckets up to 60s, and then 60+ since 60s is default for bazel.

Reply:
Updated example config with infinity bucket. Also in the example I have used different latency buckets for server and workers as developers can have different latency bucket for server and worker.